omnibus_build.sh

```diff
@@ -1,27 +1,19 @@
 #!/bin/bash -e
-PROJECT_DIR=~vagrant/dd-agent-omnibus
+PROJECT_DIR=dd-agent-omnibus
 PROJECT_NAME=datadog-agent
-
-echo "Show exported defaults"
-cat /etc/profile.d/vagrant.sh
+${LOG_LEVEL:=info}
 
 # Clean up omnibus artifacts
-sudo rm -rf /var/cache/omnibus/pkg/*
-sudo rm -rf /tmp/pip_build_vagrant
+rm -rf /var/cache/omnibus/pkg/*
 
 # Clean up what we installed
-sudo rm -f /etc/init.d/datadog-agent
-sudo rm -rf /etc/dd-agent
-sudo rm -rf /opt/$PROJECT_NAME/*
+rm -f /etc/init.d/datadog-agent
+rm -rf /etc/dd-agent
+rm -rf /opt/$PROJECT_NAME/*
 
 cd $PROJECT_DIR
 # Install the gems we need, with stubs in bin/
-su vagrant -c "bundle install --binstubs"
-su vagrant -c "bundle update" # Make sure to update to the latest version of omnibus-software
-su vagrant -c "./bin/omnibus build -l=$LOG_LEVEL $PROJECT_NAME"
-
-# TODO: add rpm signing
-#if [ #{ENV['PKG_TYPE']} == "rpm" ] && [ #{ENV['GPG_PASSPHRASE']} ] && [ #{ENV['GPG_KEY_NAME']} ]; then
-#    chmod +x rpm-sign
-#    su vagrant -c "./rpm-sign #{ENV['GPG_KEY_NAME']} #{ENV['GPG_PASSPHRASE']} pkg/#{project_name}-#{ENV['AGENT_VERSION']}-#{ENV['BUILD_NUMBER']}.*.rpm"
-#fi
+/bin/bash -l -c "git pull"
+/bin/bash -l -c "rvm use 2.2.2"
+/bin/bash -l -c "bundle update" # Make sure to update to the latest version of omnibus-software
+/bin/bash -l -c "bin/omnibus build -l=$LOG_LEVEL $PROJECT_NAME"
```

Comment:
Can't we just source the rvm script once and avoid to launch multiple new shells from scratch?